tests/test_functors.py

```diff
@@ -305,6 +313,65 @@ def testLocalPhotometry(self):
             df[("meas", "HSC-G", "base_LocalPhotoCalib")],
             df[("meas", "HSC-G", "base_LocalPhotoCalibErr")])
 
+        self.assertTrue(np.allclose(nanoJansky.values,
+                                    flux * calib,
+                                    atol=1e-13,
+                                    rtol=0))
+        self.assertTrue(np.allclose(mag.values,
+                                    (flux * calib * u.nJy).to_value(u.ABmag),
+                                    atol=1e-13,
+                                    rtol=0))
+        self.assertTrue(np.allclose(nanoJanskyErr.values,
+                                    np.hypot(fluxErr * calib, flux * calibErr),
+                                    atol=1e-13,
+                                    rtol=0))
+        self.assertTrue(np.allclose(
+            magErr.values,
+            2.5 / np.log(10) * nanoJanskyErr.values / nanoJansky.values,
+            atol=1e-13,
+            rtol=0))
+
+        # Test functors
+        func = LocalNanojansky("base_PsfFlux_instFlux",
```

Comment:
Why not have one test case for each class instead of putting everything into a very long method? You'll get more information when you run the tests, it'll be easier to make changes to the test code, and you won't have to worry about weird side effects from shared variables like `func` and `val`.

Reply:
Moved the tests into their own function so the duplicated code is reduced. Considering how closely related the functors are and how they dependent they are the previously tested methods, I'm fine with leaving them here for now.